### PR TITLE
Registry bugs

### PR DESCRIPTION
**Pull request by ptdatta**
**Notes for Reviewers**

1. Doted line.
2. Search bar.
3. Added loader.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MeshModelRegistry/MesheryTreeView.js [hunk 1]
```diff
@@ -110,10 +110,10 @@ const MesheryTreeView = ({
 
   const handleScroll = (scrollingView) => () => {
     const div = event.target;
-    if (div.scrollTop >= div.scrollHeight - div.clientHeight - 2) {
+    if (div.scrollTop >= div.scrollHeight - div.clientHeight - 10) {
       setPage((prevPage) => ({
-        ...prevPage, // Keep the current values for other keys
-        [scrollingView]: prevPage[scrollingView] + 1, // Increment the specific key based on the view
+        ...prevPage,
```
Comment (leecalcote) [inline ui/components/MeshModelRegistry/MesheryTreeView.js:115]: We lost a comment, @ptdatta. Please re-comment the code.

Comment (ptdatta) [reply ui/components/MeshModelRegistry/MesheryTreeView.js]: Done.✅

**Comment by sudhanshutech** [comment]
@ptdatta merge conflict...

**Comment by sudhanshutech** [comment]
please balance this extra space
image

**Comment by sudhanshutech** [comment]
ignoring mobile reponsiveness as its in another pr , looking good to me

**Comment by sudhanshutech** [comment]
@ptdatta can you share a screen recording?

**Comment by sudhanshutech** [comment]
search should be in same row
image

**Comment by sudhanshutech** [comment]
@ptdatta check each element at every width to confirm everything looks good(consider upto 375px atleast)
- use max-width and max-height
- use flex,  and grid element from mui they ensure responsive automatically rather hardcoding width and height

**Comment by sudhanshutech** [comment]
@ritiksaxena124 yo have any suggestion to add for mobile view.  // @ptdatta please provide a screen recording to else can review

**Comment by leecalcote** [comment]
Merge conflict

**Comment by sudhanshutech** [comment]
@ptdatta would you please post left over feedback or items so we can keep track, i made another pr with few more refactoring and enhancements so lets lose this

**Pull request closed without merging**